public/app/app-init.js

```
@@ -0,0 +1,613 @@
+/**
+ * Creates (or reuses) the AudioContext and loads the AudioWorklet module.
+ * Must be called from a user-gesture handler (click) for autoplay policy.
+ * Safe to call multiple times — no-ops if already initialized.
+ */
+async function ensureAudioContext() {
+  if (audioCtx && audioCtx.state !== 'closed') {
+    if (audioCtx.state === 'suspended') await audioCtx.resume();
+    return;
+  }
+
+  audioCtx = new AudioContext({ sampleRate: 48000, latencyHint: 'interactive' });
```

ESLint

[error] 'AudioContext' is not defined.

(no-undef)